src/Microsoft.PackageManagement.ArchiverProviders/project.json

```diff
@@ -44,8 +60,7 @@
         "netstandard1.6": {
             "imports": ["dnxcore50"],
             "buildOptions": {
-                "define": ["CORECLR"],
-                "debugType": "portable"
+                "define": ["CORECLR"]
```

Comment:
A question, before this change, the `.pdb` produced for CoreCLR are `portable`, but this setting doesn't affect building target `.net451`. After the change, I guess the setting `"debugType": "portable"` will also apply to full clr building, right? If so, what does that imply? Does the portable `pdb` work for FullCLR powershell?